Eastern-Research-Group/mywaterway
# Feature/767 simplify loading s3 files

## Related Issues:
* [HMW-767](https://jira.epa.gov/browse/HMW-767)

## Main Changes:
* Moved to a single endpoint for loading s3 files (like EQ). We were loading most files on every page, so we weren't saving much by loading them separately. The main reason for doing this was to reduce the number of logs we were getting, because people were trying to hack the app by passing garbage urls into the proxy.
  * Note: The proxy code is still there, but only for running in the local environment. This is mainly for cyan, because there cors rules don't allow localhost.

## Steps To Test:
1. Navigate to http://localhost:3000/community/harrisburg%20pa/overview
2. Verify the app loads
3. Verify the list has a waterbody without GIS in it
4. Test saving to ArcGIS Online
5. Test as many other features as possible

## TODO:
- [ ] Verify google analytics is operating correctly after this is merged into Dev.
